src/materialized/src/lib.rs

```diff
@@ -193,6 +193,9 @@ pub enum SecretsControllerConfig {
         /// The name of a Kubernetes context to use, if the Kubernetes configuration
         /// is loaded from the local kubeconfig.
         context: String,
+        user_defined_secret: String,
+        user_defined_secret_mount_path: String,
+        coordd_pod_name: String,
```

Comment:
We don't use the name "coordd" in this repository. (I'd like to shift the other repository to use "materialized" instead of "coordd" to reduce the confusion.) I suggest generalizing this argument to sidestep the problem and calling it "refresh_pod_names" or somesuch, where it takes an arbitrary list of pods whose annotations should be refreshed.

```suggestion
        coordd_pod_name: String,
```